int32 frame_id
bool is_body_move
bool  is_pwoffsw